libraries/chain/include/eosio/chain/snapshot_scheduler.hpp

```diff
@@ -38,10 +38,19 @@ class snapshot_scheduler {
    struct snapshot_request_information {
       uint32_t block_spacing = 0;
       uint32_t start_block_num = 0;
-      uint32_t end_block_num = 0;
+      uint32_t end_block_num = UINT32_MAX - 1;
```

Comment:
Why `-1` ? Seems like `std::numeric<uint32_t>::max()` would be a better value.

Reply:
Changed, thank you! (-1 was because I had comparison with end_block_num+1 in the code, changed that as well)